lib/ansible/parsing/splitter.py

```diff
@@ -68,33 +68,7 @@ def parse_kv(args, check_raw=False):
             else:
                 raise
 
-        raw_params = []
-        for orig_x in vargs:
-            x = _decode_escapes(orig_x)
-            if "=" in x:
-                pos = 0
-                try:
-                    while True:
-                        pos = x.index('=', pos + 1)
-                        if pos > 0 and x[pos - 1] != '\\':
-                            break
-                except ValueError:
-                    # ran out of string, but we must have some escaped equals,
-                    # so replace those and append this to the list of raw params
-                    raw_params.append(x.replace('\\=', '='))
-                    continue
-
-                k = x[:pos]
-                v = x[pos + 1:]
-
-                # FIXME: make the retrieval of this list of shell/command
-                #        options a function, so the list is centralized
-                if check_raw and k not in ('creates', 'removes', 'chdir', 'executable', 'warn'):
```

Comment:
This doesn't actually address this `FIXME`.  

The `FIXME` indicates that `('creates', 'removes', 'chdir', 'executable', 'warn')` should be moved somewhere more global.

This change refactors one function, into 2, which is not bad, but doesn't satisfy the `FIXME`.